.travis.yml

```diff
@@ -1,10 +1,64 @@
 language: php
-         
+
+sudo: required
```

Comment:
Failed because Travis CI is disallowing many packages we are trying to use to build libvips (only `libpango1.0-dev` is installed). So I think sudo is required.